### PR TITLE
Use dedicated IAM role for platform services

### PR DESCRIPTION
**Pull request by rajeevr2715**
### Requirements

Replace the iam user role with platform role for create the subscription filters and loggroup, swagger files, Generate the swagger files etc

### Description of the Change

As part of IAM role split, replace the iam user role with platform role for create the subscription filters and loggroup, swagger files, Generate the swagger files etc

### Benefits

Basic lambda execution role with less policies

### Possible Drawbacks

NA

### Applicable Issues

NA

### CODE REVIEW

File: builds/jenkins-build-pack-api/Jenkinsfile [hunk 1]
```diff
@@ -736,7 +736,7 @@ def createSubscriptionFilters(stackName, region, roleId) {
 	} catch (Exception ex) { }// ignore error if not created yet
 
 	try {
-		sh "aws logs put-subscription-filter --output json --log-group-name \"${lambda}\" --filter-name \"${lambda}\" --filter-pattern \"\" --destination-arn \"${logStreamer}\" --region ${region} --role-arn \"arn:aws:iam::${roleId}:role/${configLoader.INSTANCE_PREFIX}_basic_execution\""
+		sh "aws logs put-subscription-filter --output json --log-group-name \"${lambda}\" --filter-name \"${lambda}\" --filter-pattern \"\" --destination-arn \"${logStreamer}\" --region ${region} --role-arn \"arn:aws:iam::${roleId}:role/${configLoader.INSTANCE_PREFIX}_platform_services\""
```
Comment (devsatishm) [inline builds/jenkins-build-pack-api/Jenkinsfile]: Can we not use `configLoader.AWS.PLATFORMSERVICES_ROLEID` directly? 
```suggestion
sh "aws logs put-subscription-filter --output json --log-group-name \"${lambda}\" --filter-name \"${lambda}\" --filter-pattern \"\" --destination-arn \"${logStreamer}\" --region ${region} --role-arn configLoader.AWS.PLATFORMSERVICES_ROLEID
```

Comment (devsatishm) [reply builds/jenkins-build-pack-api/Jenkinsfile]: Same with `jenkins-build-pack-lambda`.

Comment (rajeevr2715) [reply builds/jenkins-build-pack-api/Jenkinsfile]: Done.

Comment (rajeevr2715) [reply builds/jenkins-build-pack-api/Jenkinsfile]: Done

**Pull request merged**